versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-commands/register.mdx

```diff
@@ -21,15 +21,15 @@ The following are the `kubeslice-cli register` command options.
 The **worker** is the resource type used with the register command.
 
 For more information on resource configuration parameters, see 
-[Kubeslice Configuration Parameters](/versioned_docs/version-0.4.0/reference/configuration-parameters). 
+[Kubeslice Configuration Parameters](/versioned_docs/version-0.4.0/reference/configuration-parameters.mdx). 
 
 ## Examples
 The following are the example commands:
 :::info
 You must switch the context to the controller cluster to register the worker cluster.
 :::
 
-1. To register a worker cluster with the KubeSlice Controller, use the following command:
+1. To register a worker cluster with the KubeSlice Controller in demo set up, use the following command:
```

Comment:
set up is action 

here it is `in the demo setup `

Reply:
updated